src/Sylius/Component/Core/Calculator/ProductVariantPriceCalculatorInterface.php

```diff
@@ -22,4 +22,9 @@ interface ProductVariantPriceCalculatorInterface
      * @throws MissingChannelConfigurationException when price for given channel does not exist
      */
     public function calculate(ProductVariantInterface $productVariant, array $context): int;
+
```

Comment:
this is a BC Break.

Reply:
That's true, we cannot merge this PR due to this change. Maybe just a separate service could do the job? Or a new one `ProductVariantPricesCalculatorInterface` (with s) and this one should be deprecated but used internally to ensure proper calculation logic